### DISCUSSION TITLE
no suitable connection found with IKEv2 policy

### DISCUSSION THREAD

**Discussion by aabababb** [strongswan/strongswan#2106] [Q&A]
conn IKEV2-ios
    keyexchange=ikev2
    ike=aes128-sha256-ecp256!
    esp=aes128-sha256!
    authby=secret
    auto=add
    left=%defaultroute
    leftid=test.net
    leftcert=server.test.net-cert.pem
    leftsendcert=always
    right=%any
    rightid=%any
    rightauth=eap-mschapv2
    eap_identity=%identity
    dpdaction=clear

Feb 18 00:34:32 test3 pluto[25291]: packet from 1.2.3.4:29684: initial parent SA message received on 10.0.0.60:500 but no suitable connection found with IKEv2 policy
Feb 18 00:34:32 test3 pluto[25291]: packet from 1.2.3.4:29684: responding to SA_INIT message (ID 0) from 1.2.3.4:29684 with unencrypted notification NO_PROPOSAL_CHOSEN

ConfiguredIKEv2 , but still unable to connect，any help?

**Comment by tobiasbrunner** [comment]
Since you have a **pluto** daemon running, which was removed from strongSwan with 5.0, you are obviously not using strongSwan but libreswan or Openswan. So please use their respective support channels.
